ui/docker/UI.Dockerfile

```diff
@@ -27,6 +27,12 @@ RUN corepack enable
 WORKDIR /app
 COPY . .
 
+ENV VITE_API_URL="UI_API_URL_PLACEHOLDER"
+ENV VITE_AUTHORITY="UI_AUTHORITY_PLACEHOLDER"
+ENV VITE_BASEPATH="UI_BASEPATH_PLACEHOLDER"
+ENV VITE_CLIENT_ID="UI_CLIENT_ID_PLACEHOLDER"
+ENV VITE_UI_URL="UI_URL_PLACEHOLDER"
```

Comment:
As each `ENV` instruction creates a new intermediate layer, would it make sense to use the `ENV` syntax that takes multiple key-value pairs here, like

```
ENV VITE_API_URL="UI_API_URL_PLACEHOLDER" \
    VITE_AUTHORITY="UI_AUTHORITY_PLACEHOLDER" \
    VITE_BASEPATH="UI_BASEPATH_PLACEHOLDER" \
    VITE_CLIENT_ID="UI_CLIENT_ID_PLACEHOLDER" \
    VITE_UI_URL="UI_URL_PLACEHOLDER"
```